### PR TITLE
Update tags v2

### PR DESCRIPTION
**Pull request by maya**
Removes some unneeded mixins from tags.

### CODE REVIEW

File: src/stylesheets/elements/_tags.scss [hunk 1]
```diff
@@ -1,11 +1,10 @@
 .usa-tag {
-  @include u-bg('base-dark');
   @include u-font('sans', '2xs');
-  @include u-margin-right(0.5);
-  @include u-padding-x(1);
-  @include u-padding-y(1px);
   @include u-radius('sm');
```
Comment (maya) [inline src/stylesheets/elements/_tags.scss]: Should this be `border-radius: units(1px);` or the mixin?

Comment (maya) [inline src/stylesheets/elements/_tags.scss]: Should this be `border-radius: units(2px);` or the mixin?

Comment (maya) [reply src/stylesheets/elements/_tags.scss]: Is there a border radius function a la `border-radius: radius(sm);`?

Comment (thisisdano) [reply src/stylesheets/elements/_tags.scss]: no, but there should be. made an issue here: https://github.com/uswds/uswds/issues/2723

Comment (maya) [reply src/stylesheets/elements/_tags.scss]: Added the function d0a8c88, seems to work.

**Pull request merged**